src/overlays/actors/ovl_En_Suttari/z_en_suttari.c

```diff
@@ -516,8 +516,8 @@ void func_80BAB434(EnSuttari* this) {
 }
 
 void EnSuttari_GetPaths(EnSuttari* this, PlayState* play) {
-    this->paths[0] = SubS_GetPathByIndex(play, ENSUTTARI_GET_PATH_INDEX(&this->actor), PATH_INDEX_MAX);
-    this->paths[1] = SubS_GetPathByIndex(play, this->paths[0]->additionalPathIndex, PATH_INDEX_MAX);
+    this->paths[0] = SubS_GetPathByIndex(play, ENSUTTARI_GET_PATH_INDEX(&this->actor), ENSUTTARI_PATH_INDEX_NONE);
+    this->paths[1] = SubS_GetPathByIndex(play, this->paths[0]->additionalPathIndex, ENSUTTARI_PATH_INDEX_NONE);
```

Comment:
```suggestion
    this->paths[0] = SubS_GetPathByIndex(play, ENSUTTARI_GET_PATH_INDEX(&this->actor), ENSUTTARI_PATH_INDEX_NONE);
    //! @bug PATH_INDEX_NONE should be (u8)PATH_INDEX_NONE
    this->paths[1] = SubS_GetPathByIndex(play, this->paths[0]->additionalPathIndex, ENSUTTARI_PATH_INDEX_NONE);
```

However, it'd be cleaner to keep a `PATH_INDEX_END` defined as 0xFF to avoid, I meant that you needed to keep the end of the list of `Path`s defined as 0xFF